README.md

```diff
@@ -433,6 +428,48 @@ func LogResponse() autorest.RespondDecorator {
 }
 ```
 
+## Tracing and Metrics
+
+All packages and the runtime are instrumented using [OpenCensus](https://opencensus.io/).
+
+### Enable
+
+As of now, tracing is disabled by default. There are 2 ways to enable tracing:
+
+- set the environment variable `AZURE_SDK_TRACING_ENABELD` (_Recommended_)
+- alternatively, import the `github.com/Azure/go-autorest/tracing` package and call the `tracing.Enable()` function or `tracing.EnableWithAIForwarding()` if using the [App Insights Forwarder](https://docs.microsoft.com/en-us/azure/application-insights/opencensus-local-forwarder).
+
+**Note**: In future major releases of the SDK, tracing may become enabled by default.
+
+### Usage
+
+Once enabled, all SDK calls will emit traces and metrics if you are instrumenting your code and the traces will correlate the SDK calls with the raw http calls made to Azure API's. If are not doing it yet, to instrument your code see how to do it with [Azure App Insights](https://docs.microsoft.com/en-us/azure/application-insights/opencensus-local-forwarder) or with [Zipkin](https://opencensus.io/quickstart/go/tracing/#exporting-traces).
```

Comment:
I think this paragraph needs to be reworked a bit.  The way it's worded makes it sound like you have to instrument your code in order to get traces and metrics which I didn't think was the case.

Reply:
hmm well you need to have an exporter registered

Reply:
i can be more explicit about it, instead of instrument, ill say have an exporter registered